modules/trusted-firmware-m/Kconfig

```diff
@@ -15,6 +15,20 @@
 
 if BUILD_WITH_TFM
 
+config TFM_CRYPTO_BUILTIN_KEYS
+	bool
+	prompt "TF-M crypto builtin keys" if !TFM_PROFILE_TYPE_MINIMAL
```

Comment:
Nit 2: Maybe we can just depends on !TFM_PROFILE_TYPE_MINIMAL then? To having two ifs?

Reply:
Yeah, I can do that also.